tests/test_git.py

```diff
@@ -607,3 +607,30 @@ def test_checkout_subdir(tmp_dir: TmpDir, scm: Git, git: Git):
     with (tmp_dir / "dir").chdir():
         git.checkout(rev)
         assert not (tmp_dir / "dir" / "bar").exists()
+
+
+@pytest.mark.skip_git_backend("pygit2", "gitpython")
+def test_describe(tmp_dir: TmpDir, scm: Git, git: Git):
+    tmp_dir.gen({"foo": "foo"})
+    scm.add_commit("foo", message="foo")
+    rev_foo = scm.get_rev()
+
+    tmp_dir.gen({"foo": "bar"})
+    scm.add_commit("foo", message="bar")
+    rev_bar = scm.get_rev()
+
+    assert scm.describe(rev_foo, "refs/heads") is None
```

Comment:
```suggestion
    assert git.describe(rev_foo, "refs/heads") is None
```

All of the `describe` calls in this test (the specific backend behavior we want to test), should use the `git` fixture instead of `scm`.

Reply:
The other usages of `scm` are fine, but in this case, using `git.<func>` ensures that we are only testing a specific backend's version of `<func>` (and not falling back to an alternate backend implementation)